sdf/1.11/mesh_shape.sdf

```diff
@@ -1,5 +1,12 @@
 <element name="mesh" required="0">
   <description>Mesh shape</description>
+
+  <attribute name="simplification" type="string" default="" required="0">
```

Comment:
I'm trying to decide if `simplification` is the right name for this attribute.

Alternatives (which might not be better):

* `//mesh/@transformation`
* `//mesh/@convexity`

with candidate values:

* `use_convex_hull`
* `use_convex_decomposition`

Reply:
I think `transformation` is generic and would be another good candidate. So we don't limit this to just creating convex meshes